Defs/Drugs/Psychite_Psycho.xml

```diff
@@ -0,0 +1,309 @@
+    <isBad>false</isBad>
+    <comps>
+      <li Class="HediffCompProperties_SeverityPerDay">
+        <severityPerDay>-2.5</severityPerDay>
+        <showHoursToRecover>true</showHoursToRecover>
+      </li>
+    </comps>
+      <stages>
+        <li>
+          <label>crash</label>
+          <hediffGivers>
+            <li Class="HediffGiver_Random">
+              <hediff>PsychoCrash</hediff>
+              <mtbDays>0.05</mtbDays>
+              <partsToAffect>
+                <li>Brain</li>
+              </partsToAffect>
+            </li>
+          </hediffGivers>
+        </li>
+        <li>
+		  <minSeverity>0.05</minSeverity>
+          <label>coming down</label>
+          <painFactor>0.75</painFactor>
+          <restFallFactor>0.75</restFallFactor>
+          <capMods>
+            <li>
+              <capacity>Consciousness</capacity>
+              <offset>0.05</offset>
+            </li>
+            <li>
+              <capacity>Sight</capacity>
+              <offset>0.05</offset>
+            </li>
+            <li>
+              <capacity>Moving</capacity>
+              <offset>0.1</offset>
+            </li>
+          </capMods>
+          <statFactors>
+            <Suppressability>0.75</Suppressability>
+          </statFactors>
+        </li>
+        <li>
+          <minSeverity>0.15</minSeverity>
+          <label>full-on high</label>
+          <painFactor>0</painFactor>
+          <restFallFactor>0.33</restFallFactor>
+          <capMods>
+            <li>
+              <capacity>Consciousness</capacity>
+              <offset>0.20</offset>
+            </li>
+            <li>
+              <capacity>Sight</capacity>
+              <offset>0.20</offset>
+            </li>
+            <li>
+              <capacity>Moving</capacity>
+              <offset>0.33</offset>
+            </li>
+          </capMods>
+          <statFactors>
+            <Suppressability>0</Suppressability>
+          </statFactors>
+        </li>
+        <li>
+          <minSeverity>0.9375</minSeverity>
+          <label>kicking in</label>
+          <painFactor>0.5</painFactor>
+          <restFallFactor>0.5</restFallFactor>
+          <capMods>
+            <li>
+              <capacity>Consciousness</capacity>
+              <offset>0.1</offset>
+            </li>
+            <li>
+              <capacity>Sight</capacity>
+              <offset>0.1</offset>
+            </li>
+            <li>
+              <capacity>Moving</capacity>
+              <offset>0.2</offset>
+            </li>
+          </capMods>
+          <statFactors>
+            <Suppressability>0.5</Suppressability>
+          </statFactors>
+        </li>
+      </stages>
+  </HediffDef>
+
+  <ThoughtDef>
+    <defName>PsychoHigh</defName>
+    <workerClass>ThoughtWorker_Hediff</workerClass>
+    <hediff>PsychoHigh</hediff>
+    <validWhileDespawned>true</validWhileDespawned>
+    <stages>
+      <li>
+        <label>coming down from psycho</label>

```

Comment:
If it doesn't have an actual mood impact, why make this a visible stage rather than just hiding it?

Reply:
It's just a flavor text to let the player know of the things that might come.